src/zfm20/zfm20.cxx

```diff
@@ -540,10 +439,10 @@ uint8_t ZFM20::deleteDB()
   return rPkt[9];
 }
 
-uint8_t ZFM20::search(int slot, uint16_t *id, uint16_t *score)
+uint8_t ZFM20::search(int slot, uint16_t &id, uint16_t &score)
 {
```

Comment:
I use typemaps to handle the references.  The wrapped method takes 1 argument and returns a list (not a tuple).  Details below.

```
/* Make it easy to use the methods which take a uint& id and score */
%include "typemaps.i"
%apply uint16_t &OUTPUT {uint16_t &id};
%apply uint16_t &OUTPUT {uint16_t &score};
```

Generates CXX with uint16_t temp variables for the two references (that I've defined at OUTPUT). 
 These temp variables get appended to the pyObject return:

```
SWIGINTERN PyObject *_wrap_ZFM20_search(PyObject *SWIGUNUSEDPARM(self), PyObject *args) {
...
uint16_t temp3 ;
uint16_t temp4 ;
...
arg3 = &temp3;
arg4 = &temp4;\
...
result = (uint8_t)(arg1)->search(arg2,*arg3,*arg4);
...
if (SWIG_IsTmpObj(res3)) {
  resultobj = SWIG_Python_AppendOutput(resultobj, SWIG_From_unsigned_SS_short((*arg3)));
...
if (SWIG_IsTmpObj(res4)) {
  resultobj = SWIG_Python_AppendOutput(resultobj, SWIG_From_unsigned_SS_short((*arg4)));
```

Which is wrapped like so in pyupm_zfm20.py:

```
def search(self, slot):
    return _pyupm_zfm20.ZFM20_search(self, slot)
```

Usage:

```
>>> import pyupm_zfm20
>>> x = pyupm_zfm20.ZFM20(0)
>>> x.setupTty(57600)
True
>>> x.generateImage()
0
>>> x.image2Tz(1)
0
>>> x.generateImage()
0
>>> x.image2Tz(2)
0
>>> x.search(1)
[0, 1, 78]

```